cookbook/profiler/data_collector.rst

```diff
@@ -46,106 +46,151 @@ access to in local properties.
-        <span class="icon"><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABoAAAAcCAQAAADVGmdYAAAAAmJLR0QA/4ePzL8AAAAJcEhZcwAACxMAAAsTAQCanBgAAAAHdElNRQffAxkBCDStonIVAAAAGXRFWHRDb21tZW50AENyZWF0ZWQgd2l0aCBHSU1QV4EOFwAAAHpJREFUOMtj3PWfgXRAuqZd/5nIsIdhVBPFmgqIjCuYOrJsYtz1fxuUOYER2TQID8afwIiQ8YIkI4TzCv5D2AgaWSuExJKMIDbA7EEVhQEWXJ6FKUY4D48m7HYU/EcWZ8JlE6qfMELPDcUJuEMPxvYazYTDWRMjOcUyAEswO+VjeQQaAAAAAElFTkSuQmCC" alt=""/></span>
-        <span class="sf-toolbar-status">Example</span>
+            <span class="icon"><img src="..." alt=""/></span>
+            <span class="sf-toolbar-status">Information</span>
         {% endset %}
 
         {% set text %}
-        <div class="sf-toolbar-info-piece">
-            <b>Quick piece of data</b>
-            <span>100 units</span>
-        </div>
-        <div class="sf-toolbar-info-piece">
-            <b>Another quick thing</b>
-            <span>300 units</span>
-        </div>
+            <div class="sf-toolbar-info-piece">
+                <b>Quick piece of data</b>
+                <span>100 units</span>
+            </div>
+            <div class="sf-toolbar-info-piece">
+                <b>Another piece of data</b>
+                <span>300 units</span>
+            </div>
         {% endset %}
 
-        {# Set the "link" value to false if you do not have a big "panel"
-           section that you want to direct the user to. #}
-        {{ include('@WebProfiler/Profiler/toolbar_item.html.twig', { 'link': true }) }}
+        {# the 'link' value set to 'false' means that this panel doesn't show
+           a section in the web profiler. #}
+        {{ include('@WebProfiler/Profiler/toolbar_item.html.twig', { link: false }) }}
+    {% endblock %}
+
+The ``toolbar`` block is used to create a panel in the web debug toolbar. The
+inner ``icon`` block defines the content to display right inside the toolbar
+and the ``text`` block defines the content displayed when hovering the mouse
+over that same panel.
+
+.. tip::
+
+    Built-in collector templates define all their images as embedded base64-encoded
+    images. This makes them work everywhere without having to mess with web assets
+    links:
+
+    .. code-block:: html
+
+        <img src="data:image/png;base64,..." />
+
+    You are encouraged to use this same technique for your own toolbar panels.
+
+If the toolbar panel includes extended web profiler information, the Twig template
+must also define other blocks:
 
+.. code-block:: jinja
+
+    {% extends '@WebProfiler/Profiler/layout.html.twig' %}
+
+    {% block toolbar %}
+        {% set icon %}
+            <span class="icon"><img src="..." alt=""/></span>
+            <span class="sf-toolbar-status">Information</span>
+        {% endset %}
+
+        {% set text %}
+            <div class="sf-toolbar-info-piece">
+                {# ... #}
+            </div>
+        {% endset %}
+
+        {# the 'link' value is now set to 'true', which allows the user to click
```

Comment:
You should addd a note sayign it is the default, and so you don't need to change the value of the variable here